packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -228,8 +228,14 @@ export const getOpenPositionsColumnsTemplate = (currency: string) => [
         title: localize('Indicative profit/loss'),
         col_index: 'profit',
         renderCellContent: ({ row_obj }: TCellContentProps) => {
-            if (!row_obj.profit_loss && (!row_obj.contract_info || !row_obj.contract_info.profit)) return;
-            const profit = row_obj.profit_loss || row_obj.contract_info.profit;
+            if (
+                !row_obj.profit_loss &&
+                row_obj.profit_loss !== 0 &&
+                !row_obj.contract_info?.profit &&
+                row_obj.contract_info?.profit !== 0
+            )
```

Comment:
@kate-deriv this is not an equivalent.
Please keep the part which ensures that if `row_obj.contract_info` is allowed to be `undefined` if `!row_obj.profit_loss` is `truthy`, and as a result we return (stop execution).
For example, like this:
```suggestion
            const { profit_loss, contract_info } = row_obj;
            if (
                !profit_loss &&
        profit_loss !== 0 &&
        (!contract_info || (!contract_info?.profit && contract_info?.profit !== 0))
            )
```

Reply:
I think if `contract_info`  is `undefined`, then `!row_obj.contract_info?.profit` and `row_obj.contract_info?.profit !== 0` will return `true`. Anyway, will apply your suggestions

Reply:
@kate-deriv you're right, please keep your variant :)